R/calc_views.R

```diff
@@ -135,6 +137,18 @@ set_view_lim <- function(views){
       usr.lim <- views[[as.numeric(i)]][['window']][[lim.name]][1:2]
       views[[as.numeric(i)]][['window']][[lim.name]] <- data.lim
       views[[as.numeric(i)]][['window']][[lim.name]][!is.na(usr.lim)] <- usr.lim[!is.na(usr.lim)]
+      
+      usr.axs <- data[[paste0(var, 'axs')]][[as.numeric(i)]]
+      view.index <- which(!names(views[[as.numeric(i)]]) %in% c('window', 'gsplot'))[which(!is.na(usr.axs))]
+      
+      if (any(!is.na(usr.axs)) && usr.axs[which(!is.na(usr.axs))] == 'o') {
```

Comment:
@ldecicco-USGS @jread-usgs can you review this and help me make it less messy? There's so much indexing/subsetting craziness